shell/components/ResourceDetail/index.vue

```diff
@@ -193,6 +193,9 @@ export default {
           opt:  { watch: true }
         });
       } catch (e) {
+        if (e.status === 404) {
+          store.dispatch('loadingError', new Error(`Resource ${ resource } with id ${ fqid } not found, unable to display resource details`));
```

Comment:
This will redirect us away from the page, so no point continuing.

We also don't need `notFound` and anything that depends on it.

Reply:
I cannot stop the execution of the template with an early return, otherwise it gives me some errors and won't redirect to the `fail-whale` page. I tried "pushing" a new route (forcing navigation to that page and the outcome is the same 🤷 )

Reply:
That makes sense.

The `v-else-if="notFound"` block can still be removed though (and `IconMessage` component import) by using `<Loading v-if="$fetchState.pending || notFound" />`

Reply:
done